src/lib/pages/backoffice/Loan/LoanDetails/AvailableItems/AvailableItems.js

```diff
@@ -155,13 +167,52 @@ export default class AvailableItems extends Component {
     }
   };
 
+  updateSearchQuery = (e, elem) => {
+    const { data } = this.props;
+    this.setState({
+      query: elem.target.value,
+      filteredData: data.hits.filter(hit =>
+        hit.metadata.barcode.includes(elem.target.value)
+      ),
+    });
+  };
+
+  onSearchExecute = () => {
+    const { query } = this.state;
+    const { data } = this.props;
+    this.setState({
+      filteredData: data.hits.filter(hit =>
+        hit.metadata.barcode.includes(query)
+      ),
+    });
+  };
+
+  renderSearchBar() {
+    const { query } = this.state;
+    return (
+      <SearchBar
+        currentQueryString={query}
+        onInputChange={this.updateSearchQuery}
+        executeSearch={this.onSearchExecute}
+        placeholder="Insert barcode..."
+      />
+    );
+  }
+
   render() {
-    const { isLoading, error } = this.props;
+    const { isLoading, error, loan, data } = this.props;
+    const statesThatRenderSearchBar = invenioConfig.CIRCULATION.loanActiveStates.concat(
```

Comment:
I would display the box `Available physical copies` only for loans in `request` status? It is already with an active loan, then it should already have an item. WDYT?

Reply:
Currently the box offers the possibility to replace the current physical copy with another one available in case the item is on loan, that's why the search bar displays there too, to be able to filter in case the librarian wants to.
![image](https://user-images.githubusercontent.com/15194802/89021576-105af780-d321-11ea-8cca-a8ed353f791c.png)
If the loan is cancelled or the item is returned the box still is displayed, I think it does not give any relevant information there. IMHO we could remove it in that case. WDYT?
![image](https://user-images.githubusercontent.com/15194802/89022027-abec6800-d321-11ea-9dc2-c0a1f80db506.png)